keras_hub/src/models/mobilenet/mobilenet_backbone.py

```diff
@@ -29,39 +29,35 @@ class MobileNetBackbone(Backbone):
+            each inverted residual block for each block in the model.
+        stackwise_num_blocks: list of ints, number of inversted residual blocks
+            per block
+        stackwise_num_filters: list of list of ints, number of filters for
             each inverted residual block in the model.
-        stackwise_num_filters: list of ints, number of filters for each inverted
-            residual block in the model.
-        stackwise_kernel_size: list of ints, kernel size for each inverted
-            residual block in the model.
-        stackwise_num_strides: list of ints, stride length for each inverted
-            residual block in the model.
+        stackwise_kernel_size: list of list of ints, kernel size for each
+            inverted residual block in the model.
+        stackwise_num_strides: list of list of ints, stride length for each
+            inverted residual block in the model.
         stackwise_se_ratio: se ratio for each inverted residual block in the
             model. 0 if dont want to add Squeeze and Excite layer.
-        stackwise_activation: list of activation functions, for each inverted
-             residual block in the model.
+        stackwise_activation: list of list of activation functions, for each
+            inverted residual block in the model.
         image_shape: optional shape tuple, defaults to (224, 224, 3).
-        depth_multiplier: float, controls the width of the network.
-            - If `depth_multiplier` < 1.0, proportionally decreases the number
-                of filters in each layer.
-            - If `depth_multiplier` > 1.0, proportionally increases the number
-                of filters in each layer.
-            - If `depth_multiplier` = 1, default number of filters from the paper
-                are used at each layer.
         input_num_filters: number of filters in first convolution layer
-        output_num_filters: specifies whether to add conv and batch_norm in the end,
-            if set to None, it will not add these layers in the end.
+        output_num_filters: specifies whether to add conv and batch_norm in the
+            end, if set to None, it will not add these layers in the end.
             'None' for MobileNetV1
         input_activation: activation function to be used in the input layer
             'hard_swish' for MobileNetV3,
             'relu6' for MobileNetV1 and MobileNetV2
         output_activation: activation function to be used in the output layer
             'hard_swish' for MobileNetV3,
             'relu6' for MobileNetV1 and MobileNetV2
-        inverted_res_block: whether to use inverted residual blocks or not,
-            'False' for MobileNetV1,
-            'True' for MobileNetV2 and MobileNetV3
+        depthwise_filters: int, number of filters in depthwise separable
+            convolution layer
+        squeeze_and_excite: float, squeeze and excite ratio in the depthwise
```

Comment:
NIT: `float, squeeze and excite ratio in the depthwise layer, None, if dont want to do squeeze and excite`
==> `float, squeeze and excite ratio in the depthwise layer. Defaults to  None.`

Please update all others args to follow the same pattern